packages/web-twig/src/Resources/components/Accordion/AccordionContent.twig

```diff
@@ -0,0 +1,37 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id -%}
+{%- set _labelledById = props.labelledById -%}
+{%- set _parent = props.parent | default(null) -%}
+{%- set _isOpen = props.isOpen | default(false) | boolprop -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Collapse' -%}
+{%- set _collapseClassName = _spiritClassPrefix ~ 'Collapse__content' -%}
+{%- set _accordionClassName = _spiritClassPrefix ~ 'Accordion__content' -%}
+{%- set _isOpenClassName = _isOpen ? 'is-open' : null -%}
+
+{# Attributes #}
+{%- set _idAttr = _id ? 'id="' ~ _id ~ '"' : null -%}
+{%- set _labelledIdAttr = _labelledById ? 'aria-labelledby="' ~ _labelledById ~ '"' : null -%}
+{%- set _parentAttr = _parent ? 'data-parent="' ~ _parent ~ '"' : null -%}
```

Comment:
According to the same principle, this would be:

```suggestion
{%- set _dataParentAttr = _parent ? 'data-parent="' ~ _parent ~ '"' : null -%}
```

Reply:
fixed